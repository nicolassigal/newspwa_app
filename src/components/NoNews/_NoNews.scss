.no-news {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: calc(100vh - 90px);
    h1 {
        font-size: 24px;
        color:var(--txt-color);
    } 
}