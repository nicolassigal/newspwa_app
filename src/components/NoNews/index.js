import React from 'react';

const NoNews = () => {
    return(
        <div className="no-news">
            <h1>Sorry, no news here to show</h1>
        </div>
    )
}

export default NoNews;