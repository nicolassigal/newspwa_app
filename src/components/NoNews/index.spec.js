import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import NoNews from './index';
import { shallow } from 'enzyme';

describe('NoNews component test', () => {
  it('NoNews component should render correctly', () => {
   const wrapper = shallow(<NoNews />)
   expect(wrapper).toMatchSnapshot();
  });
});