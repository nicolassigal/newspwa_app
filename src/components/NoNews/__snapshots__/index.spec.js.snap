// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoArticle component test NoArticle component should render correctly 1`] = `ShallowWrapper {}`;
